### ISSUE TITLE
static displayName getter interferes with livereactreload

### ISSUE THREAD

**Issue by ibgreen** [uber-common/react-vis#6]
livereactreload overwrites displayName while it proxies React components. This fails since react-vis uses a static getter to define the displayName attribute. As alternative to `static get displayName()` it would be great if we could either rely on auto deduction of `displayName`s or just assign them after the class declaration:
```
class Component extends React.Component {
  ...
}
Component.displayName = 'Component';
```

**Comment by bulyonov** [comment]
Thank you, @ibgreen . This was fixed in #20 and merged into master. Will publish a new version soon.

**Comment by bulyonov** [comment]
Thank you, @ibgreen . This was fixed in #20 and merged into master. Will publish a new version soon.
